suite-native/module-assets/src/devicesSlice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { TrezorDevice } from '@suite-common/suite-types';
+
+export const actionPrefix = '@devices';
+
+export type DummyDevice = Pick<TrezorDevice, 'id' | 'state' | 'label'>;
+
+export type DevicesState = DummyDevice[];
+
+const initialState: DevicesState = [];
+
+export type DevicesRootState = {
+    devices: DevicesState;
+};
+
+const create = (state: DevicesState, payload: DummyDevice) => {
```

Comment:
it seems really uneccessary to have it in extra function because of one line of code, please place it to reducer directly